iOS/CFC_Tracker/GoogleOpenSource.framework/GoogleOpenSource

```diff
@@ -0,0 +1 @@
+Versions/Current/GoogleOpenSource
```

Comment:
Add in some argument about why you are checking in the GoogleOpenSourceFramework. The argument about the lack of versioning was particularly compelling to me.